llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -767,6 +767,142 @@ Operation *FftOp::getTiledImplementation(OpBuilder &builder, ValueRange outputs,
   return tiledFftOp;
 }
 
+//===----------------------------------------------------------------------===//
+// ScanOp
+//===----------------------------------------------------------------------===//
+
+static LogicalResult verifyScanOp(ScanOp op) {
+  if (op.inputs().size() != 1) {
```

Comment:
I think the shape of the output w.r.t input needs to be checked as well for the inclusive and exclusive cases (only when the shapes are static, dynamic case you cant really check at this level)